tests/unittest/others/test_cache_transceiver_precheck_config.py

```
@@ -298,6 +300,119 @@ def test_use_kv_cache_manager_v2_flags():
+            raise RuntimeError("model hook failed")
+
+    api = types.SimpleNamespace(TorchLlmArgs=lambda **kwargs: types.SimpleNamespace(**kwargs))
+    monkeypatch.setattr(rp, "load_internal_apis", lambda: api)
+    monkeypatch.setattr(rp, "_lookup_model_cls", lambda _model_dir: (FailingModel, object()))
+    cache_cfg = types.SimpleNamespace(transceiver_runtime="PYTHON")
+    side = {
+        "use_kv_cache_manager_v2": "auto",
+        "parallel": {"tp": 2, "pp": 1, "cp": 1},
+    }
+
+    with pytest.raises(RuntimeError, match="get_model_defaults failed.*refusing to assume V1"):
```

Ruff (0.16.1)

[warning] Pattern passed to `match=` contains metacharacters but is neither escaped nor raw

(RUF043)